cmd/cosign/webhook/main.go

```diff
@@ -115,6 +121,10 @@ func NewValidatingAdmissionController(ctx context.Context, cmw configmap.Watcher
 }
 
 func NewMutatingAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	// Decorate contexts with the current state of the config.
+	store := config.NewStore(logging.FromContext(ctx).Named("config-store"))
+	store.WatchConfigs(cmw)
+
```

Comment:
Why do we need the config store in the mutating webhook part?

Reply:
It might not be needed there, and I can yank it. I was wondering if that might be useful for some controls for things like do you resolve tags, etc. I can take it out and we can add it later if need.